pkg/client/cli/command_group.go

```diff
@@ -90,8 +79,8 @@ Examples:
 
 Available Commands:
 {{- if commandGroups .}}
-{{- range $group := commandGroups .}}
-  {{$group.Name}}:{{range $group.Commands}}
+{{- range $name, $group := commandGroups .}}
```

Comment:
Same comment as above here, it think it would be clearer to rename `$group` to `$commands`